.changes/unreleased/tinymce-TINY-11332-2024-10-08.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: 'Invalid HTML elements within an `svg` element are removed. #TINY-1132'
```

Maybe this:

```suggestion
body: 'Invalid HTML elements within SVG elements was not removed.
```

Or:
```suggestion
body: 'Invalid HTML elements within `svg` elements was not removed.
```

Or:
```suggestion
body: 'Invalid HTML elements within `svg` elements is now removed.
```

---

Updated changelog entry